python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -175,9 +175,10 @@ def op_config(self) -> Any:
 
     @property
     def dagster_run(self) -> DagsterRun:
-        """PipelineRun: The current pipeline run."""
+        """DagsterRun: The current run."""
```

Comment:
also updated this to be identical as `run` so we know they are the same and one is public and the other is not.